spec/smartcar/e2e/smartcar_spec.rb

```diff
@@ -18,11 +18,16 @@
         scopes = %w[read_odometer read_location]
 
         result = subject.get_compatibility(vin: tesla_vin, scope: scopes)
-
         expect(result.compatible).to be_truthy
+        expect(result.reason).to be_nil
+        expect(result.capabilities.length).to eq(2)
+        expect(result.capabilities.map(&:capable).inject(&:&)).to be_truthy
```

Comment:
this line is some ruby black magic :mage: `inject(&:&)` :astonished: 

I really should go learn ruby :p

Reply:
hahaha...the `&:` thingy is my favorite :D